src/sentry/event_manager.py

```diff
@@ -1867,10 +1867,11 @@ def _create_group(project: Project, event: Event, **kwargs: Any) -> Group:
 
     # get severity score for use in alerting
     if features.has("projects:first-event-severity-calculation", event.project):
```

Comment:
This is a very risky code path. Put this logic into its own function and wrap it with a try/except block.